src/components/Form/Calculator/Display/index.js

```diff
@@ -0,0 +1,13 @@
+const Display = ({borrowAmount, annualSalary, monthlyRepay}) => {
```

Comment:
You mentioned prop destructuring earlier on - this is it.

When you have a param in the component like this:

```
const Display = (props) => {}
```

React will pass in the props given to this component in the form of an object. In this example to access the three props you have you would need to do `props.borrowAmount`. 

`{borrowAmount, annualSalary, monthlyRepay}` is the destructuring syntax, it takes that object and splits it up into those three variables so you don't have to keep doing `props.`